lib/ui/page/auth/view.dart

```diff
@@ -159,7 +160,9 @@ class AuthView extends StatelessWidget {
           child: Stack(
             fit: StackFit.expand,
             children: [
-              IgnorePointer(child: ColoredBox(color: style.colors.background)),
+              if (!PlatformUtils.isWeb)
+                IgnorePointer(
+                    child: ColoredBox(color: style.colors.background)),
```

Comment:
```suggestion
                IgnorePointer(
                    child: ColoredBox(color: style.colors.background),),
```

https://dart.dev/effective-dart/style#consider-changing-your-code-to-make-it-more-formatter-friendly